connectors/sources/jira.py

```diff
@@ -466,27 +481,46 @@ async def _get_projects(self):
                 async for response in self.jira_client.api_call(url_name=PROJECT):
                     response = await response.json()
                     for project in response:
-                        yield {
-                            "_id": f"project-{project['id']}",
-                            "_timestamp": timestamp,
-                            "Type": "Project",
-                            "Project": project,
-                        }
+                        await self._put_projects(project=project, timestamp=timestamp)
             else:
                 for project_key in self.jira_client.projects:
                     async for response in self.jira_client.api_call(
                         url_name=PROJECT_BY_KEY, key=project_key
                     ):
                         project = await response.json()
-                        yield {
-                            "_id": f"project-{project['id']}",
-                            "_timestamp": timestamp,
-                            "Type": "Project",
-                            "Project": project,
-                        }
+                        await self._put_projects(project=project, timestamp=timestamp)
+            await self.queue.put("FINISHED")  # pyright: ignore
         except Exception as exception:
             logger.warning(f"Skipping data for type: {PROJECT}. Error: {exception}")
 
+    async def _get_specific_issue(self, issue):
```

Comment:
```suggestion
    async def _get_specific_issue(self, issue):
```

Should it be a `_put_issue` similar to `_put_projects`?